building/build-debs/homeworld-services/wrappers/start-worker.sh

```diff
@@ -7,6 +7,8 @@ systemctl daemon-reload
 
 systemctl start rkt-api
 systemctl enable rkt-api
+systemctl start aci-pull-monitor
+systemctl enable aci-pull-monitor
```

Comment:
Is there a reason why auth-monitor isn't enabled anywhere in the previous PR?

Reply:
[It is...?](https://github.com/sipb/homeworld/pull/204/files#diff-7ccc7d124c8952d23de333ac39b60446R130)